### PR TITLE
Pagetree styles bugs

### PR DESCRIPTION
**Pull request by vxsx**
this PR goes in hand with https://github.com/divio/djangocms-admin-style/pull/338

# before:
screen shot 2016-05-17 at 12 07 39
screen shot 2016-05-17 at 12 07 18
screen shot 2016-05-17 at 12 07 14
screen shot 2016-05-17 at 12 07 45

# after:
screen shot 2016-05-17 at 12 08 36
screen shot 2016-05-17 at 12 08 40
screen shot 2016-05-17 at 12 08 25
screen shot 2016-05-17 at 12 08 31

### CODE REVIEW

File: cms/static/cms/sass/components/pagetree/_tree.scss [hunk 1]
```diff
@@ -41,7 +41,9 @@
 // general container around the jstree or elements
 .cms-pagetree-root.filtered {
     min-height: auto;
-    box-shadow: $pagetree-box-shadow;
+    // important is to fight with admin-style
+    box-shadow: $pagetree-box-shadow !important;
+    padding: 0 !important;
```
Comment (lory87) [inline cms/static/cms/sass/components/pagetree/_tree.scss]: ordering

File: cms/static/cms/sass/components/pagetree/_tree.scss [hunk 2]
```diff
@@ -119,9 +121,12 @@
     width: 100%;
     line-height: $pagetree-header-height;
     height: $pagetree-header-height;
-    margin: 0 !important;
-    span {
-        font-weight: normal;
+    h1 {
+        margin: 0 !important;
+        line-height: $pagetree-header-height;
```
Comment (lory87) [inline cms/static/cms/sass/components/pagetree/_tree.scss]: ordering

**Comment by lory87** [comment]
👍

**Comment by Brankub** [comment]
👍

**Pull request merged**